python/CMakeLists.txt

```diff
@@ -22,6 +22,6 @@ set(OPENMMPLUMED_LIBRARY_DIR "${CMAKE_BINARY_DIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/setup.py)
 add_custom_command(TARGET PythonInstall
     COMMAND "${PYTHON_EXECUTABLE}" setup.py build
-    COMMAND "${PYTHON_EXECUTABLE}" setup.py install
+    COMMAND "${PYTHON_EXECUTABLE}" setup.py install --prefix ${CMAKE_INSTALL_PREFIX}
```

Comment:
I don't think this is correct.  The python module should go in the standard python install location, which is different from the OpenMM install location.